docker-compose.yml

```diff
@@ -341,29 +341,25 @@ services:
   #
   # IRC for FAF chat.
   #
-  faf-unrealircd:
-    container_name: faf-unrealircd
-    image: faforever/faf-unrealircd:v1.2
-    user: ${FAF_UNREALIRCD_USER}
+  faf-ircd:
+    container_name: faf-ircd
+    image: bbriggs/unrealircd
     restart: unless-stopped
-    ulimits:
-      nofile:
-        soft: 8192
-        hard: 8192
     networks:
       faf:
         aliases:
+          - "faf-ircd"
           - "faf-unrealircd"
           - "irc.faforever.com"
           - "services.faforever.com"
     depends_on:
       - faf-init-volumes
     volumes:
-      - ./config/faf-unrealircd/ssl/server.cert.pem:/home/unreal/unrealircd/conf/ssl/server.cert.pem
-      - ./config/faf-unrealircd/ssl/server.key.pem:/home/unreal/unrealircd/conf/ssl/server.key.pem
-      - ./config/faf-unrealircd/services.conf:/home/unreal/unrealircd/services/conf/services.conf
-      - ./config/faf-unrealircd/unrealircd.conf:/home/unreal/unrealircd/conf/unrealircd.conf
-      - ./config/faf-unrealircd/motd.txt:/home/unreal/unrealircd/conf/ircd.motd
+      - ./config/faf-ircd/ssl/server.cert.pem:/data/unrealircd/conf/tls/server.cert.pem
+      - ./config/faf-ircd/ssl/server.key.pem:/data/unrealircd/conf/tls/server.key.pem
+      - ./config/faf-ircd/unrealircd.conf:/data/unrealircd/conf/unrealircd.conf
+      - ./config/faf-ircd/motd.txt:/data/unrealircd/conf/ircd.motd
+      - ./data/faf-ircd/:/data/unrealircd/data/
     ports:
       - "6667:6667"
```

Comment:
Reduce to port 6667 only?

Reply:
I do agree with this, but will need also a configuration file slimming